### DISCUSSION TITLE
Trouble in running RNAlysis

### DISCUSSION THREAD

**Discussion by Rajeshwarira** [community/community#52229] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello sir/madam,
I have tried to run the data given as example in the user guide ( analysis 1) . But I was not able to run the program. Please do help me with troubleshooting the errors I attached as photo as below.
![IMG20230406160416](https://user-images.githubusercontent.com/128403613/230549606-834867a4-a2c8-4bae-9144-2c083ba2496b.jpg)
Version of my system: Windows 10

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/GuyTeichman/RNAlysis? In that case you should look for support information *in that repository*, usually in the README or a file called SUPPORT. If those don't say anything, opening an issue there is usually good. :slightly_smiling_face:

**Comment by Rajeshwarira** [comment]
Thank you for your response and I'll look into it.
